CAAppKitBridge/Demo/main.m

```diff
@@ -0,0 +1,90 @@
+/* Demo/main.m
+
+   Copyright (C) 2018 Free Software Foundation, Inc.
+
+   Author: Stjepan Brkic <stjepanbrkicc@gmail.com>
+   Date: June 2018
+
+   This file is part of QuartzCore.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; see the file COPYING.LIB.
+   If not, see <http://www.gnu.org/licenses/> or write to the
+   Free Software Foundation, 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110-1301, USA.
+*/
+
+#import "../Source/GSCAData.h"
+#import "../Source/NSView+CAMethods.h"
+#import <Foundation/Foundation.h>
+
+@interface DemoController : NSObject
```

Comment:
It is a convention to put classes into their own `h`eader and i`m`plementation files. I suppose `DemoController` that you provided in these commits is actually an outdated version and should be replaced with the implementation in `main.m`?

Reply:
Exactly. Should I delete it?

Reply:
I left it here because at some point you mentioned that it may come handy later, and told me to keep it tracked

Reply:
You should decide which implementation is correct, and that implementation should live in its own files named `DemoController.m` and `DemoController.h`.